stats/thresholds.go

```diff
@@ -83,7 +83,7 @@ func (t *Threshold) Run() (bool, error) {
 }
 
 type Thresholds struct {
-	Runtime    *goja.Runtime
+	runtime    *goja.Runtime
```

Comment:
While I agree with this change, it's out of scope for this PR. Please submit it as another one.

Reply:
@liclac the reason for making the `Runtime` not exported is because `gob` serialization throw an error since `goja.Runtime` have no exported member. It prints the error to the stdout otherwise.

Instead of writing a MarshalBinary I decided to make this member not as exported.

Reply:
Ah, I see.